core/circuitbreaker/slot.go

```diff
@@ -31,10 +31,11 @@ func (b *Slot) Check(ctx *base.EntryContext) *base.TokenResult {
 		return result
 	}
 	if passed, rule := checkPass(ctx); !passed {
+		msg := "circuit breaker check not pass"
```

Comment:
Maybe "circuit breaker check  blocked" is better?

Reply:
Updated.